src/main.rs

```diff
@@ -187,6 +187,29 @@ fn get_package_count_debian_based() -> Result<String> {
     Ok(pkg)
 }
 
+fn get_package_count_void() -> Result<String> {
+    let xbps = Command::new("xbps-query").arg("-l").output().context(Pkgcount)?;
+    let pkgs = bytecount::count(&xbps.stdout, b'\n');
+    let pkg = format!("{}", pkgs);
+    Ok(pkg)
+}
+
+fn get_package_count_fedora() -> Result<String> {
+    let dnf = Command::new("dnf").arg("list").arg("--installed").output().context(Pkgcount)?;
+    let pkgs = bytecount::count(&dnf.stdout, b'\n');
+    let pkgs = pkgs as u32 - 1;
+    let pkg = format!("{}", pkgs);
+    Ok(pkg)
+}
+
+fn get_package_count_pip() -> Result<String> {
+    let pip = Command::new("pip").arg("list").output().context(Pkgcount)?;
+    let pkgs = bytecount::count(&pip.stdout, b'\n');
+    let pkgs = pkgs as u32 - 2;
```

Comment:
Same

Reply:
I tried to do it without them and cargo threw errors at me about how it's unable to do binary operations on strings. So I made them u32 and it worked.

Reply:
The previous value is an `usize` as returned by `bytecount::count`, so it should work.

Reply:
That's really weird. I swear I tried that before and it didn't work. But it's working now, so I'll edit and commit again.